### PR TITLE
OCPBUGS#35232: Moving vSphere vols not supported

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-35232

Link to docs preview: https://78114--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#persistent-storage-csi-vsphere-limitations_persistent-storage-csi-vsphere

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @gcharot @duanwei33 @gnufied

### CODE REVIEW

File: modules/persistent-storage-csi-vsphere-limitations.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-limitations_{context}"]
+= vSphere CSI limitations
+
+The following limitations apply to the vSphere Container Storage Interface (CSI) Driver Operator:
+
+* The vSphere CSI Driver supports dynamic and static provisioning. However, when using static provisioning in the PV specifications, do not use the key `storage.kubernetes.io/csiProvisionerIdentity` in `csi.volumeAttributes` because this key indicates dynamically provisioned PVs.
```
Comment (gnufied) [inline modules/persistent-storage-csi-vsphere-limitations.adoc:12]: Where is this `csiProvisionerIdentity` text coming from?

Comment (michaelryanpeter) [inline modules/persistent-storage-csi-vsphere-limitations.adoc:12]:
```suggestion
* The vSphere CSI Driver supports dynamic and static provisioning. However, when using static provisioning in the persistent volume (PV) specifications, do not use the key `storage.kubernetes.io/csiProvisionerIdentity` in `csi.volumeAttributes`. This key indicates dynamically provisioned PVs.
```
Two nits. The second sentence was getting a little long to me. Feel free to disregard if you don't find it useful

**Review by michaelryanpeter — COMMENTED** [review]
Two suggestions to consider. LGTM.

/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed

File: modules/persistent-storage-csi-vsphere-limitations.adoc (hunk 1 above)
Comment (lpettyjo) [reply modules/persistent-storage-csi-vsphere-limitations.adoc:12]: That is existing content. I just moved it into this new Limitations section.

**Comment by gcharot** [comment]
LGTM

**Pull request merged**